lib/terminal.robot

```diff
@@ -153,8 +153,10 @@ Execute Command In Terminal
     Set DUT Response Timeout    ${timeout}
     IF    '${DUT_CONNECTION_METHOD}' == 'Telnet'
         ${output}=    Telnet.Execute Command    ${command}    strip_prompt=True
-    ELSE IF    '${DUT_CONNECTION_METHOD}' == 'SSH'
-        ${output}=    SSHLibrary.Execute Command    ${command}
+        # With this executing the command via ssh to not return any
+        # output
+        # ELSE IF    '${DUT_CONNECTION_METHOD}' == 'SSH'
```

Comment:
Won't this break multiple tests which use SSH?

Reply:
This change is something like a cherry-pick-hot-fix of this change https://github.com/Dasharo/open-source-firmware-validation/pull/455/files which is now merged into develop. I will remove it so it merges well.